src/components/Dropdown/PrioritySelector/PrioritySelector.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+import Close from '@material-design-icons/svg/round/close.svg';
+import KeyboardDoubleArrowUp from '@material-design-icons/svg/round/keyboard_double_arrow_up.svg';
+import KeyboardArrowUp from '@material-design-icons/svg/round/keyboard_arrow_up.svg';
+import KeyboardArrowDown from '@material-design-icons/svg/round/keyboard_arrow_down.svg';
+import { bem } from '../../../utils/bem';
+import { Dropdown } from '../Dropdown';
+import { ListItem } from '../../List';
+import { Text } from '../../Text';
+import styles from './PrioritySelector.scss';
+import { IconButton } from '../../Buttons';
+
+const iconMap = {
+    mandatory: KeyboardDoubleArrowUp,
+    important: KeyboardArrowUp,
+    optional: KeyboardArrowDown,
+    exclude: Close,
+};
+
+export type Priority = 'mandatory' | 'important' | 'optional' | 'exclude';
+
+export type PriorityItem = {
```

Comment:
can you add docs (comments) for what each means? I think value is the most obscure.